qml/pages/FirstPage.qml

```diff
@@ -41,67 +62,91 @@ Page {
             }
         }
 
-        SilicaListView {
-            id: listView
-            model: []
-            anchors.fill: parent
-            header: PageHeader {
-                title: qsTr("Current Broadcasts")
+        onModelChanged: rowCount = Math.ceil(model.length / columnCount)
+
+        // Cell width and height depend on columnCount, which depends on orientation
+        cellWidth: page.width / columnCount
+        cellHeight: page.width / columnCount / 16.0 * 9.0 + Theme.fontSizeMedium + Theme.paddingMedium
+
+        // Always, always remember to include the header itemi height...
```

Comment:
Fix typo or remove whole comment if not necessary or maybe open with the comment `why` this needs to be remembered? :)

Reply:
That was really a note to self - I always, always forget that... Removing the comment.